src/bindings/python/src/openvino/frontend/tensorflow/node_decoder.py

```diff
@@ -152,7 +152,8 @@ def get_attribute(self, name):
             if self.m_parsed_content.size == 1:
                 if isinstance(self.m_parsed_content, np.ndarray):
                     return OVAny(Tensor(self.m_parsed_content))
-                return OVAny(Tensor(np.array([self.m_parsed_content]), shape=[1]))
+                self.m_parsed_content = np.array(self.m_parsed_content)
+                return OVAny(Tensor(self.m_parsed_content))
             ov_tensor = Tensor(self.m_parsed_content, shared_memory=self.m_shared_memory)
```

Comment:
Are we ready to remove the skip here: https://github.com/openvinotoolkit/openvino/blob/master/tests/model_hub_tests/tf_hub_tests/precommit_models#L9?

Or is rootcause different?

Reply:
Rootcause is different, mobilenet_v1_100_224 still fails.